src/components/modals/ChallengeModal.tsx

```diff
@@ -84,16 +87,18 @@ export const ChallengeModal = (props: Props) => {
         <EdgeText style={styles.titleText} numberOfLines={2}>
           {lstrings.complete_captcha_title}
         </EdgeText>
-        <EdgeTouchableOpacity
-          style={styles.closeIconContainer}
-          onPress={handleCancel}
-        >
-          <AntDesignIcon
-            name="close"
-            color={theme.primaryText}
-            size={theme.rem(1.25)}
-          />
-        </EdgeTouchableOpacity>
+        {!isDesktop ? null : (
+          <EdgeTouchableOpacity
+            style={styles.closeIconContainer}
+            onPress={handleCancel}
+          >
+            <AntDesignIcon
+              name="close"
+              color={theme.primaryText}
+              size={theme.rem(1.25)}
+            />
+          </EdgeTouchableOpacity>
+        )}
```

Comment:
It looks like the goal is to only show the close button on desktop? However, I think we need a close button on mobile too, since there is no other way to escape. The challenge HTML eats the scroll gesture, so swipe-to-close doesn't work, and there's no other exit button. Maybe we can just drop the commit.

Reply:
Dropped the commit